library/webpack-plugins/chunk-manifest-plugin.js

```diff
@@ -0,0 +1,45 @@
+/*
+  This plugin listens for the `chunk-asset` hook and records the following information:
+
+  {
+    'chunkName': {
+      filename: 'public/path/to/chunk',
+      hasRuntime: true/false,
+      parents: ['chunkName1', ..., 'chunkNameN']
+    }
+  }
+
+  It supplies a `chunk-manifest` hook that can be used by other plugins to obtain this information.
+
+  On top of that it generates a `chunk-manifest.json` file that can be used to render the correct
+  scripts on a page.
+*/
+
+const { RawSource } = require('webpack-sources')
+
+module.exports = chunkManifestPlugin
```

Comment:
why not `module.exports = function chunkManifestPlugin () { .. }`?

Reply:
Great point, I should be more consistent with the plugins :+1: